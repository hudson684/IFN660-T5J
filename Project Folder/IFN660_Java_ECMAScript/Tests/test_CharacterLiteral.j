'a'
'b'
'\r'
'\7'
'\17'
'\217'
'\555'