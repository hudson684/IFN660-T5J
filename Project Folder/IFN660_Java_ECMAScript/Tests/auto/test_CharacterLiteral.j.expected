CharacterLiteral ('\'')
CharacterLiteral ('a')
CharacterLiteral ('%')
CharacterLiteral ('\t')
CharacterLiteral ('\\')
CharacterLiteral ('\'')
CharacterLiteral ('\u03a9')
CharacterLiteral ('\uFFFF')
CharacterLiteral ('\177')
CharacterLiteral ('T')
EOF